katka/serializers.py

```diff
@@ -41,16 +41,11 @@ class EmbeddedTeamSerializer(serializers.Serializer):
     slug = serializers.SlugField(required=False)
 
 
-class ProjectSerializer(serializers.ModelSerializer):
-    team = EmbeddedTeamSerializer(required=False, read_only=True)
-
-    class Meta:
-        model = Project
-        fields = ('public_identifier', 'slug', 'name', 'team')
-
+class TeamChildMixin:
     def to_internal_value(self, data):
         # Can't put this on the EmbeddedTeamSerializer because it will never be called (team is not required and
         # read-only, so it's not allowed to pass the team when modifying something).
+        data = super().validate(data)
```

Comment:
Hm, shouldn't this be `super().to_internal_value(data)` instead?

Reply:
Yes it should. Changing this actually broke the test in a place where I wondered why it didn't break before. I forgot to come back to that, so I'm glad I fixed this.